### ISSUE TITLE
Documentation missing for commit message Closing keywords

### ISSUE THREAD

**Issue by erikmd** [git4school/git4school-visu#73]
**Description**

I've recently tested Git4School for preparing some M1 GitHub assignment (thanks for developing this tool!), and it took me a long time to find the exact syntax of the commit message closing syntax.

(All the more as last week, the question chooser interface for selecting question markers was not that intuitive… but I've just seen that you enhanced the corresponding message in the meantime *via* [`283b7410`](https://github.com/git4school/git4school-visu/commit/283b7410e025e0dafaeb49aa13a63ae2ceec10cb#diff-7164935394a83166925ce8964f8c1cf5c9979487b07f98dadf67e941403292fbR258) :+1: :)

**Steps to reproduce**

On https://git4school.github.io/git4school-visu/index.html it is only written:

> Filter commits by TP group or by *resolved question* […]

without further details.

**Expected info**

I was not sure whether the syntax was similar to [that of GitHub](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), but I was able to found the snippet at stake:
https://github.com/git4school/git4school-visu/blob/ddff00b77db776ce6bef08fee3831f29b8a540d2/src/app/models/Commit.model.ts#L160-L177

So I guess it'd be worth to document it, with some regexp or just sentences recapitulating these keywords…

To sum up, if I'm not mistaken, Git4School accepts all of these closing sentences (for each supported keyword `Resolve` etc.):

1. `Resolve ex1: random text`, `Resolve ex1 random text` (with a colon or a space) or simply `Resolve ex1`
2. `Resolve #1: random text`, `Resolve #1 random text` (with a colon or a space) or simply `Resolve #1`

and the main "advantage" of 2. items would be that the question markers `#n` can at the same time be issues numbers that the assignment topic could ask the students to create… if the teacher wishes it.

**Comment by F0urchette** [comment]
Hi, **thanks** for the issue ! There was indeed a lack of information on that side. The migration of the user documentation has just been done #76. On this occasion, the documentation has been reworked and I was able to integrate a note on the closing keywords [here](https://git4school.github.io/docs/conventions.html#closing-commit-keywords).

**Comment by erikmd** [comment]
Hi @F0urchette, OK thanks! so I guess I can close the issue :)
